patches/minecraft/net/minecraft/enchantment/EnchantmentHelper.java.patch

```diff
@@ -1,6 +1,22 @@
 --- ../src-base/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
 +++ ../src-work/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
-@@ -302,7 +302,7 @@
+@@ -143,11 +143,12 @@
+         return field_77520_b.field_77497_a;
+     }
+ 
+-    public static float func_152377_a(ItemStack p_152377_0_, EnumCreatureAttribute p_152377_1_)
```

Comment:
Probably leave it before subclasses and after all methods (in a forge block is better)